### PR TITLE
Tc 45 jest tests part 3

### PR DESCRIPTION
**Pull request by rizwan171**
Another batch of jest tests.

To test:
1. Run `yarn test`
2. Confirm all tests pass

### CODE REVIEW

File: src/components/NavBar/BoardTitle/BoardTitle.tsx [hunk 1]
```diff
@@ -38,7 +38,7 @@ const BoardTitle = () => {
         <input
           type="text"
           value={editableTitle}
-          id="rounded-email"
```
Comment (rizwan171) [inline src/components/NavBar/BoardTitle/BoardTitle.tsx:41]: This id wasn't being used for anything

File: src/tests/components/CardModal/CardModalActions/CopyMenu/CopyMenu.test.tsx [hunk 2]
```diff
@@ -98,10 +98,6 @@ describe("CopyMenu", () => {
 
     fireEvent.click(copyButton);
 
-    await waitFor(() => {
-      expect(store.getState()).not.toEqual(stateBeforeCopy);
-    });
```
Comment (rizwan171) [inline src/tests/components/CardModal/CardModalActions/CopyMenu/CopyMenu.test.tsx:103]: This waitFor was unnecessary as no async actions happen when you copy a card

File: src/tests/components/CardModal/CardModalActions/DeleteMenu/DeleteMenu.test.tsx [hunk 3]
```diff
@@ -72,10 +72,6 @@ describe("DeleteMenu", () => {
 
     fireEvent.click(deleteButton);
 
-    await waitFor(() => {
-      expect(store.getState()).not.toEqual(stateBeforeDelete);
-    });
```
Comment (rizwan171) [inline src/tests/components/CardModal/CardModalActions/DeleteMenu/DeleteMenu.test.tsx:77]: This waitFor was unnecessary as no async actions happen when you move a card

File: src/tests/components/CardModal/CardModalActions/MoveMenu/MoveMenu.test.tsx [hunk 4]
```diff
@@ -100,10 +100,6 @@ describe("MoveMenu", () => {
 
     fireEvent.click(moveButton);
 
-    await waitFor(() => {
-      expect(store.getState()).not.toEqual(stateBeforeMove);
-    });
```
Comment (rizwan171) [inline src/tests/components/CardModal/CardModalActions/MoveMenu/MoveMenu.test.tsx:105]: This waitFor was unnecessary as no async actions happen when you move a card

**Review by nk711 — APPROVED** [review]
Works as expectsed all tests passes.

**Pull request merged**